spreadsheets/collision_rank.yaml

```diff
@@ -0,0 +1,943 @@
+# reserved for UX/UI
+- _reserved: {from: 1, to: 100}
+# reserved for house theme overlays
+- _reserved: {from: 101, to: 201}
+# reserved for data overlays
+- _reserved: {from: 202, to: 302}
+- {layer: 'earth', kind: 'continent'}
```

Comment:
`$layer` instead, since `layer` is a valid (OSM) property?

Reply:
Good point. Fixed in 87241c9.